src/Object/Api/Mastodon/Account.php

```diff
@@ -113,9 +112,9 @@ public function __construct(BaseURL $baseUrl, array $publicContact, Fields $fiel
 
 		$this->note            = BBCode::convert($publicContact['about'], false);
 		$this->url             = $publicContact['url'];
-		$this->avatar          = (($userContact['photo'] ?? '') ?: $publicContact['photo']) ?: DI::baseUrl() . '/photo/contact/'. (($userContact['id'] ?? 0) ?: $publicContact['id']);
+		$this->avatar          = (($userContact['photo'] ?? '') ?: $publicContact['photo']) ?: Contact::getAvatarForContact(($userContact['id'] ?? 0) ?: $publicContact['id']);
 		$this->avatar_static   = $this->avatar;
-		$this->header          = DI::baseUrl() . '/photo/header/'. (($userContact['id'] ?? 0) ?: $publicContact['id']);
+		$this->header          = Contact::getHeaderForContact(($userContact['id'] ?? 0) ?: $publicContact['id']);
```

Comment:
This can be simplified further, adding the suggested new method name
```suggestion
		$this->header          = Contact::getHeaderUrlForId($userContact['id'] ?? 0 ?: $publicContact['id']);
```